jib-core/src/test/java/com/google/cloud/tools/jib/http/WithServerConnectionTest.java

```diff
@@ -52,4 +56,22 @@ public void testErrorOnSecondSend() throws IOException, InterruptedException {
       }
     }
   }
+
+  @Test
+  public void testInsecureConnection()
+      throws IOException, InterruptedException, GeneralSecurityException, URISyntaxException {
+    try (TestWebServer server = new TestWebServer(true);
+        Connection connection =
+            new Connection.Builder(new URL(server.getEndpoint()))
+                .doNotValidateCertificate()
+                .build()) {
+      Response response = connection.send("GET", new Request.Builder().build());
+
+      Assert.assertEquals(200, response.getStatusCode());
+
+      ByteArrayOutputStream out = new ByteArrayOutputStream();
+      response.getBody().writeTo(out);
+      Assert.assertEquals("Hello World!", out.toString("UTF-8"));
```

Comment:
Should prob use `StandardCharsets.UTF_8` rather than the string form.

Reply:
`toString()` doesn't have the version to accept `Charset` directly, but I can do `StandardCharsets.UTF_8.name()`.

Reply:
In other parts of the codebase, we've been using like `new String(out.toByteArray(), StandardCharsets.UTF_8)`